app/decorators/session_decorator.rb

```diff
@@ -37,6 +37,10 @@ def mfa_expiration_interval
 
   def failure_to_proof_url; end
 
+  def opt_out_rem_me
```

Comment:
So this means that signing in without an SP can use "remember me" right? I think it would benefit from a name change, so `disable_remember_me: false` would be clear that this does not disable remember me.

or if we went the other way: `allow_remember_me: true`  could be clearer?

Reply:
Disable remember me (to me) would apply we are disabling the control which is not the intent.

Reply:
Ok what about `default_remember_me_value: false`

Reply:
Sure.  I guess i'll take it a step further and call it `default_remember_device_value`

Reply:
actually not crazy about value.  How about `default_remember_device`

Reply:
`default_remember_device` works for me!